src/components/menu-bar/project-title-input.css

```diff
@@ -15,6 +15,8 @@ $title-width: 12rem;
     width: $title-width;
     height: auto;
     padding: .5rem;
+    flex-grow: 1;
+    width: 2rem;
```

Comment:
I think it is debatable if even `width` should be controlled by the component (rather than the parent).

Reply:
Good point I will investigate!